src/functions/instance/getTableData.js

```diff
@@ -2,6 +2,26 @@ import describeTable from '../api/instance/describeTable';
 import searchByValue from '../api/instance/searchByValue';
 import searchByConditions from '../api/instance/searchByConditions';
 
+const getAttributesFromTableData = (tableData, existingAttributes) => {
+  const existing = new Map(existingAttributes.map((value, index) => [value, index]));
+  const extra = new Map();
+  for (const dataRow of tableData) {
+    for (const key of Object.keys(dataRow)) {
+      if (!existing.has(key)) {
+        const count = extra.get(key) || 0;
+        extra.set(key, count + 1);
+      }
+    }
+  }
+  const result = [];
+  for (const [key, count] of extra) {
+    if (count >= 8) {
```

Comment:
This doesn't do the same thing I had suggested with my code. What I want:
* Dynamic attributes should only be added if they result in a maximum of eight columns
* Include dynamic attributes with the maximum count of records, up to the eight column maximum.

There is nothing about this that has anything to do with the record count being more or less than 8.

Reply:
I interpreted it as creating a character map and only displaying key/value pairs of dynamically added columns if there 8+ instances of that key defined throughout the loaded table rows